cpuid/src/lib.rs

```diff
@@ -5,7 +5,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the THIRD-PARTY file.
 
-#![warn(missing_docs)]
+#![deny(missing_docs)]
```

Comment:
To fix the #944 issue, can you also replace warn with deny in the other crates as well?

Reply:
Done